src/x509/clu_request_setup.c

```diff
@@ -262,9 +261,42 @@ static int _wolfSSL_X509_extensions_print(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
 static int _wolfSSL_X509_REQ_attributes_print(WOLFSSL_BIO* bio,
         WOLFSSL_X509* x509, int indent)
 {
-    (void)bio;
-    (void)x509;
-    (void)indent;
+    WOLFSSL_X509_ATTRIBUTE* attr;
+    char scratch[MAX_WIDTH];
+    int i = 0;
+
+    XSNPRINTF(scratch, MAX_WIDTH, "%*s%s", indent, "", "Attributes: \n");
+    if (wolfSSL_BIO_write(bio, scratch, (int)XSTRLEN(scratch)) <= 0) {
+        return WOLFSSL_FAILURE;
+    }
+
+    attr = wolfSSL_X509_REQ_get_attr(x509, i);
```

Comment:
I was thinking you could literally put the assignment in the `while ( )` condition but this is good enough.